### ISSUE TITLE
Build system is broken

### ISSUE THREAD

**Issue by asyropoulos** [fontforge/fontforge#2306]
Hello,

I am working on OpenIndiana and I have compiled many times fontoforge with no problem. Today I tried to compile the latest version of FontForge and I did a git clone to get the latest source. Initially, I did an autoreconf but the system reported that certain things where missing and that there are no lib and libltdl directories. An automake solved some problem but I had to manually create the directories. Finally, the system created a configure file which had a number of  offensive commands:

gl_MANYWARN_ALL_GCC([warnings])

  # Enable all GCC warnings not in this list.
  gl_MANYWARN_COMPLEMENT([warnings, $warnings, $nw])
  for w in $warnings; do
    gl_WARN_ADD([$w])
  done

  # Add an extra warning
  gl_WARN_ADD(-Wstrict-overflow=1)

  # Add these explicitly: adding -Wfoo to $nw doesn't work in this case
  gl_WARN_ADD([-Wno-unused-result]) # FIXME: We should in fact use this, but currently it makes too much noise
  gl_WARN_ADD([-Wno-comment]) # We use /* in comments with good reason

  # Add some more safety measures
  gl_WARN_ADD([-D_FORTIFY_SOURCE=2])
  gl_WARN_ADD([-fmudflap])

I was getting an error message about a offensive left parenthesis. After I deleted these commands
(they produce warnings...) I rerun configure and no it stops with the following message

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named IPython
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: error: cannot find input file: `Makefile.in'

**Comment by jtanx** [comment]
It sounds like you didn't run bootstrap. You shouldn't be calling autoreconf/automake/libtoolize directly, but doing:

> ./bootstrap

You can find full build instructions [here](https://github.com/fontforge/fontforge/blob/master/INSTALL-git.md).

**Comment by asyropoulos** [comment]
>
>It sounds like you didn't run bootstrap. You shouldn't be calling autoreconf/automake/libtoolize directly, but doing:
>./bootstrap
>You can find full build instructions here.

Indeed this solved my problem. However, compilation stopped when trying to 

generate files unistd.h and wchar.h. I tried to build with make V=1 and then 

I copied the code that appeared on the screen in a file. I manually run the 

file and it did create these files. Apart from this I did not encounter any 

other problem.

A.S.
 
----------------------
Apostolos Syropoulos
Xanthi, Greece
